examples/scripts/my_test_facts.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+# Copyright: (c) 2020, Your Name <YourName@example.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: my_test_facts
+
+short_description: This is my test facts module
+
+version_added: "2.11"
+
+description: This is my longer description explaining my test info module.
```

Comment:
```suggestion
description: This is my longer description explaining my test facts module.
```
Do you mean facts module?